content/en/docs/Tasks/update-and-ship-operator.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Update and ship an Operator"
+date: 2021-03-10
+weight: 2
+description: > 
+    In depth explanation of operator update and shipment process. 
+--- 
+
+## Introduction
+
+In the Operator Lifecycle Manager (OLM) ecosystem, the following resources are used to resolve Operator installations and upgrades:
```

Comment:
See my suggestions in : https://github.com/operator-framework/olm-docs/pull/136/files